### PR TITLE
Avatar updates

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: app-typescript/components/Avatar.tsx [hunk 1]
```diff
@@ -12,7 +13,15 @@ interface IImageAvatar extends IBase {
 interface ITextAvatar extends IBase {
     text: string; // limited to 3 characters
 }
-
+interface INumberAvatar extends IBase {
+    number: string;
+}
+interface IIconAvatar extends IBase {
+    status?: 'default' | 'in-progress' | 'done';  // defaults to default
+    type?: 'text' | 'photo' | 'video' | 'audio' | 'file' | 'post' | 'post' | 'list-alt' | 'graphic';
```
Comment (MarkLark86) [inline app-typescript/components/Avatar.tsx:21]: What is the benefit here of restricting what icons can be used with an Avatar?
I know we don't have other use cases right now, but would it be beneficial to allow any icon here (and remove the `canceled` prop, and the Planning module will choose which icon based on desired coverage assignment state)?

File: app-typescript/components/Avatar.tsx [hunk 2]
```diff
@@ -12,7 +13,15 @@ interface IImageAvatar extends IBase {
 interface ITextAvatar extends IBase {
     text: string; // limited to 3 characters
 }
-
+interface INumberAvatar extends IBase {
+    number: string;
+}
+interface IIconAvatar extends IBase {
+    status?: 'default' | 'in-progress' | 'done';  // defaults to default
```
Comment (MarkLark86) [inline app-typescript/components/Avatar.tsx:20]: Same with `status`, can be simply use the same `type` from the `Icon` component (and let the Planning module choose which colour to apply?)

**Comment by tomaskikutis** [comment]
It should have been marked as WIP @MarkLark86. Fritz made the UI and I'll later review and fix the structure/interfaces.

**Comment by tomaskikutis** [comment]
continued work in https://github.com/superdesk/superdesk-ui-framework/pull/758 I didn't push changes to this PR in order not to mess up Fritz's `develop` branch.

**Pull request closed without merging**